app/src/main/kotlin/com/hedvig/app/feature/claims/ui/ClaimsUtil.kt

```diff
@@ -0,0 +1,27 @@
+package com.hedvig.app.feature.claims.ui
+
+import android.content.Context
+import android.content.Intent
+import androidx.fragment.app.FragmentManager
+import com.hedvig.app.feature.claims.ui.pledge.HonestyPledgeBottomSheet
+import com.hedvig.hanalytics.HAnalytics
+
+suspend fun startClaimsFlow(
```

Comment:
This seems to be called only from 1 place if I see correctly, maybe just move it as a private function there?

Reply:
Its called from common claims and from home